Restore Windows build-ability

DavidLudwig opened the pull request:
This restores the ability to build for MS Windows. It was done as part of a [re]port of Executor 2000 to Windows, which has a separate PR at autc04/executor#17

DavidLudwig committed:
windows build fixes

autc04 commented:
I don't get this change.. what went wrong with writing a `std::string` to `std::cout`?

DavidLudwig commented:
The overload for operator<<(basic_ostream<...> &, const string &) couldn't be found.

Looking this over again, it looks like a different, workable fix would be to add '#include <string>' to the list of #includes.  I'll modify this PR to do just that.

DavidLudwig committed:
re-fix build error involving std::string

operator<<(basic_ostream &, const string &) are declared in <string>, at
least in Microsoft's C++ std-lib implementation (included as part of
Visual Studio).

autc04 commented:
Right. The compiler would have been correct in rejecting it outright with that include missing. My bad :-)
